scripts/build-module.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/node
+
+'use strict'
+
+const path = require('path')
+const glob = require('glob')
+const fs = require('fs')
+
+const babel = require('babel-core')
+const outputFileSync = require('output-file-sync')
+
+let options = JSON.parse(require('fs').readFileSync('.babelrc', 'utf8'))
```

Comment:
oh, didn’t know about that. sure!

so it would be:

```json
{
  "env": {
    "development": {"presets": ["es2015-loose"]},
    "build-module": {"presets": ["es2015-loose-rollup"]}
  },
  "presets": ["stage-0", "react"],
  "plugins": ["transform-decorators-legacy"]
}
```

because "development" is the default.

only remaining question: does this work or do we need to do this?

```json
{
  "env": {
    "development": {
      "presets": ["es2015-loose", "stage-0", "react"]
    },
    "build-module": {
      "presets": ["es2015-loose-rollup", "stage-0", "react"]
    }
  },
  "plugins": ["transform-decorators-legacy"]
}
```